l1-contracts/scripts/register-hyperchain.ts

```diff
@@ -66,7 +66,7 @@ async function main() {
     .option("--validium-mode")
     .option("--base-token-name <base-token-name>")
     .option("--base-token-address <base-token-address>")
-    .option("--use-governance <use-governance>")
+    .option("---avoid-governance <avoid-governance>")
```

Comment:
1. triple `---`
2. As stated in the previous comment, it is better to keep the old way as it is more readable